tox.ini

```diff
@@ -23,13 +23,14 @@ passenv =
 setenv =
     COVERAGE_FILE = {toxworkdir}/.coverage.{envname}
     TEST_STATUS_DIR = {envtmpdir}
+    PYPY3323BUG = 1
     path: TEST_MODE = path
     sdist: TEST_MODE = sdist
     wheel: TEST_MODE = wheel
 extras =
     test
 commands =
-    pytest -rsx --cov --cov-config pyproject.toml \
+    pytest -ra --cov --cov-config pyproject.toml \
```

Comment:
What does`-a` do?

Reply:
Right, that's the value of `-r`.  I would've separated them with a space for clarity, otherwise it looks like you're chaining multiple flags.